.prettierignore

```diff
@@ -2,3 +2,4 @@
 node_modules
 **/node_modules
 **/lib
+.vscode
```

Comment:
You shouldn't need the two `node_modules`, and I don't see a problem with formatting the `.vscode` folder.